tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -597,6 +597,17 @@ TEST_P(CPUOnly, groupConvTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// This test targets the DKKC8 optimization.
+TEST_P(CPUOnly, nonSquarePaddingConvTest) {
+  std::array<size_t, 4> S{{1, 2, 1, 128}};
+  Tensor out1(ElemKind::FloatTy, S);
+  Tensor out2(ElemKind::FloatTy, S);
+  inferNonSquarePaddingConv(&out1, backendKind_);
```

Comment:
Let's be explicit and put BackendKind::CPU here, code will be more readable this way.